Changelogs for Digital Twin Manager UI 1023.0.7 (#4684)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1023-3-4-upgrade-web-sdk-version-to-1023-14-148.md

---
date: ""

> title: "Upgrade webSDK version to 1023.14.148"

Reviewer: ```suggestion
title: "Upgraded Web SDK version in DTM to 1023.14.148"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"

> ticket: ""

Reviewer 2: ```suggestion
ticket: "CTM-2893"
```

version: "1023.3.4"
---

> The Digital Twin Manager application has been upgraded to the Cumulocity Web SDK version 1023.14.148 to address styling related issues.

Reviewer: ```suggestion
The Digital Twin Manager application has been upgraded to the {{< product-c8y-iot >}} Web SDK version 1023.14.148 to address styling-related issues.
```
